tox.ini

```diff
@@ -62,7 +62,7 @@ commands =
 [testenv:flake8]
 basepython = python3
 deps =
-    flake8
+    flake8==4.0.1
```

Comment:
Why are we pinning the flake8 version here? `4.0.1` seems to be very old. 
https://pypi.org/project/flake8/#history

Reply:
The latest flake8 throws a lot formatting errors even on files this PR does not touch. Downgrading to 4.0.1 (which i think is the last merged PR's version) resolved the issues. Looks like there are some formatting rule changes that breaks our check.